sample/SampleHost/Program.cs

```diff
@@ -19,6 +21,24 @@ static void Main(string[] args)
                 config.UseDevelopmentSettings();
             }
 
+            // Build up a LoggerFactory to log to App Insights, but only if this key exists.
+            string instrumentationKey = Environment.GetEnvironmentVariable("ApplicationInsightsInstrumentationKey");
```

Comment:
Application Insights will use this env var: [APPINSIGHTS_INSTRUMENTATIONKEY](https://github.com/Microsoft/ApplicationInsights-dotnet/blob/42d178c83d0cc72fa5270448e14e2044f1a790bf/src/Core/Managed/Net40/Extensibility/Implementation/TelemetryConfigurationFactory.cs#L22) Also the same is used by Web Apps (actually they came up with the name